README.md

```diff
@@ -13,41 +13,54 @@ GLUON_SITE_FEEDS='community'
 
 PACKAGES_COMMUNITY_REPO=https://github.com/freifunk-gluon/community-packages.git
 PACKAGES_COMMUNITY_COMMIT=<COMMIT-HASH>
+PACKAGES_COMMUNITY_BRANCH=master
 ```
 
-## Commit guidelines
+If you leave out the `_BRANCH`, the repository default branch is used - your commit has to be on the specified branch.
+
+You should always test and check changes before deploying a new commit, as many people of the community have commit rights and there is currently no test or release process defined.
+
+## Contribution guidelines
+
+### Commit guidelines
 
 * The master branch is protected against the use of `git push --force`.
-* Use Pull Requests if you are uncertain about your patches.
+* Use Pull Requests if you are uncertain about your patches - this is generally a good practice to get feedback from others.
 * Merge commits are disabled in the master branch. If you accept single-commit pull-requests, do a "rebase"- or "squash"-merge instead.
   This will prevent the commit-history to be cluttered with superfluous "merge" logmessages.
+  You can also merge with fast-forward locally and push this commit on the master branch, to keep the same commit hash.
 
-## Commit-message guidelines
+### Commit-message guidelines
 
 Every commit message should prefixed with the package-name, followed by a colon.
 
-## Creating a package
+It should also contain a summary of the changes and a short reasoning why they are needed
+
+### Creating a package
 
 For a general documentation on how OpenWrt packages software, see this wiki entry.
 
 https://openwrt.org/docs/guide-developer/packages
 
 Each package has it's own subdirectory in the community-repository.
 
-### Package naming
+#### Package naming
 
 Ideally, the name consists of the maintaining communities short-handle, followed by the package name.
 
 After this scheme, a community with the short-handle `ffap` would name their package `ffap-sample-package`.
 
-### PKG_LICENSE
+It is also possible to add packages with the `ff` slug to generalize the ownership or add packages without a tying maintenance to a specific community.
+
+#### PKG_LICENSE
 
 The PKG License should be defined as a SPDX ID. See the SPDX FAQ for more details.
 
 [spdx.dev/learn/handling-license-info](https://spdx.dev/learn/handling-license-info/#how)
 
+It is mandatory for new packages to include a license in the Makefile
```

Comment:
Maybe change the `FantasyLicense` in the example below to a proper license to make it easier understanding the SPDX headers.